src/DependencyInjection/Container.php

```
@@ -101,6 +115,14 @@ public function hasParameter($name) {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public function setParameter($name, $value) {
+    $this->parameters[$name] = $value;
```

Again this container is always "frozen" and such should throw an exception.